src/newSeed/stage4.ts

```diff
@@ -140,6 +141,13 @@ export class Stage4 extends BaseStage {
       message = "Please enter a valid value for End Time";
     } else if (this.endDate < this.startDate) {
       message = "Please select an End Date greater than the Start Date";
+    } else if (this.endDate.getTime() === this.startDate.getTime()) {
```

Comment:
Can you not replace all this with `if (this.endDate <= this.startDate)` ?

Reply:
No, it will not work. The endDate and startDate don't have times in them.

Reply:
Sorry, I meant:  `if (this.seedConfig.seedDetails.endDate <= this.seedConfig.seedDetails.startDate)`.

Why not this?

Reply:
seedConfig's end and start Date are only set after the validation has been done and the persis data function is called. So their value is undefined at first during validation

Reply:
The nice clean way to do this would be to put this logic into a function and use it in both places.  The function would return the Date object.

```
let temp = this.startDate;
let temp = Object.assign({}, this.startDate);
temp.setHours(Number.parseInt(startTimes[0]), Number.parseInt(startTimes[1]));
```